app/services/authcore/authcore.ts

```diff
@@ -196,6 +205,37 @@ export class AuthCoreAPI {
     return signed
   }
 
+  getOfflineDirectSigner(): OfflineDirectSigner {
+    const chainId = this.cosmosChainId
+    const getAddressesAndPubKeys = this.getCosmosAddressesAndPubKeys.bind(this);
+    const sign = this.cosmosSign.bind(this);
```

Comment:
Can we use arrow function below to avoid bind() here?

Reply:
Done